modules/apps/object/object-storage-salesforce/src/main/java/com/liferay/object/storage/salesforce/internal/rest/manager/v1_0/SalesforceObjectEntryManagerImpl.java

```diff
@@ -289,6 +309,40 @@ companyId, _getDateFormat(),
 				scopeKey, search));
 	}
 
+	private ObjectField _getObjectFieldByExternalReferenceCode(
+		String externalReferenceCode1, List<ObjectField> objectFields) {
+
+		for (ObjectField objectField : objectFields) {
+			String externalReferenceCode2 =
+				objectField.getExternalReferenceCode(
+				).trim();
+
+			if (externalReferenceCode2.contains(" ")) {
+				externalReferenceCode2 = externalReferenceCode2.split(" ")[1];
+			}
+
+			if (Objects.equals(
+					externalReferenceCode1, externalReferenceCode2)) {
+
+				return objectField;
+			}
+		}
+
+		return null;
+	}
+
+	private ObjectField _getObjectFieldByName(
+		String name, List<ObjectField> objectFields) {
+
+		for (ObjectField objectField : objectFields) {
+			if (Objects.equals(name, objectField.getName())) {
+				return objectField;
+			}
+		}
+
+		return null;
+	}
```

Comment:
Hi @jpince,

These two methods (`_getObjectFieldByExternalReferenceCode` and `_getObjectFieldByName`) seems to not being used, could you please ask for someone remove it and resend?

Thank you!